src/html.rs

```diff
@@ -79,27 +87,40 @@ pub fn rewrite_html_base_url(
                 ),
                 // Rewrite background image url.
                 element!("body>div.bg-primary.text-main", |el| {
-                    if let Some(mut attr) = el.get_attribute("style") {
-                        if attr.starts_with("background-image: url('/") {
-                            let mut base_url = "";
-                            if let Some(url) = site_url {
-                                base_url = url;
+                    if let Some(style) = el.get_attribute("style") {
+                        let mut pairs = helpers::split_styles(&style);
+                        let backgrond_image_url = match pairs.get("background-image") {
+                            Some(value) if CSS_STATIC_URL_REGEX.is_match(value) => {
+                                if let Some(cdn_url) = cdn_url {
+                                    value.replacen("/static", cdn_url, 1)
```

Comment:
the replacen isn't an in-place modification, therefore it should be assigned the value

Reply:
According to the docs: `replacen creates a new String, and copies the data from this string slice into it.`, I think it works as expected.

Reply:
seems wrong: 
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f2912d0d07994a1d45915a1bf677fe47